fe/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -554,29 +555,29 @@ public boolean isNonNullResultWithNullParamFunctions(String funcName) {
     private static final Map<Type, String> BITMAP_INTERSECT_INIT_SYMBOL =
             ImmutableMap.<Type, String>builder()
                     .put(Type.TINYINT,
-                            "_ZN5doris15BitmapFunctions21bitmap_intersect_initIaN9doris_udf10TinyIntValEEEvPNS2_15FunctionContextEPNS2_9StringValE")
+                            "_ZN5doris15BitmapFunctions20intersect_count_initIaN9doris_udf10TinyIntValEEEvPNS2_15FunctionContextEPNS2_9StringValE")
```

Comment:
You can't change this name directly.
Because it will cause query fail when grayscale upgrade

Reply:
The grayscale upgrade itself will also cause the query to fail，so I think the impact is not significant.